src/Core/Framework/MessageQueue/Service/MessageSizeCalculator.php

```diff
@@ -13,6 +13,12 @@
 #[Package('framework')]
 class MessageSizeCalculator
 {
+    /**
+     * @see https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/quotas-messages.html
+     * Maximum message size is 262144 (1024 * 256) bytes
+     */
+    public const MESSAGE_SIZE_LIMIT = 1024 * 256;
```

Comment:
Relocating this doesn’t seem right to me. The responsibility of `MessageSizeCalculator` doesn't suggest any association